Src/FluentAssertions/EventRaisingExtensions.cs

```diff
@@ -53,49 +53,43 @@ public static IEventRecording WithSender(this IEventRecording eventRecording, ob
         }
 
         /// <summary>
-        /// Asserts that at least one occurrence of the events had at least one of the arguments matching a predicate. Returns
-        /// only the events that matched that predicate.
+        /// Asserts that at least one of the occurred events is of the expected type <typeparamref name="T"/>
+        /// and optionally has a matching predicate. Returns
+        /// only the events that matched those type and predicate.
```

Comment:
Some thoughts:
We don't know if any events occured.
The arguments should be of type `T`, not the event itself.
It's one the arguments that should match a predicate.

```suggestion
        /// Asserts that at least one occurence of the events had one or more arguments of the expected type <typeparamref name="T"/>
        /// any of which matched a predicate. Returns
        /// only the events that matched both type and predicate.
```